eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/RemotingHelper.java

```diff
@@ -72,13 +72,13 @@ public static String parseChannelRemoteAddr(final Channel channel) {
         return "";
     }
 
-    public static String parseSocketAddressAddr(SocketAddress socketAddress) {
+    public static String parseSocketAddressAddr(InetSocketAddress socketAddress) {
         if (socketAddress != null) {
-            final String addr = socketAddress.toString();
+            return socketAddress.getAddress().getHostAddress() + ":" + socketAddress.getPort();
 
-            if (addr.length() > 0) {
-                return addr.substring(1);
-            }
+            //if (addr.length() > 0) {
+            //    return addr.substring(1);
+            //}
```

Comment:
I think it's better to directly delete code here instead of commenting out.

Reply:
done.